elixir/apps/web/lib/web/live/resources/show.ex

```diff
@@ -113,22 +113,33 @@ defmodule Web.Resources.Show do
               Address
             </:label>
             <:value>
-              <%= @resource.address %>
+              <span :if={@resource.type == :internet}>
+                0.0.0.0/0, ::/0
```

Comment:
Should we mention this is only available to Clients/Gateways `1.2.0` here in the UI somewhere?

Reply:
@jamilbk we do that when you create a Policy for it